app/src/ui/stashing/confirm-discard-stash.tsx

```diff
@@ -0,0 +1,78 @@
+import React = require('react')
```

Comment:
:art: my favourite suggestion

```suggestion
import * as React from 'react'
```

Reply:
oh yeah we should do that